OpenRA.Game/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -224,6 +256,14 @@ public override bool HandleKeyPress(KeyInput e)
 			return false;
 		}
 
+		static IEnumerable<Actor> SelectActorsInBoxWithDeadzone(World world, int2 a, int2 b, Func<Actor, bool> cond)
+		{
+			if (a == b || (a - b).Length > 50)
```

Comment:
It might just be me, but I think this number is too high. How about 24?

If you change this, please remember to also change the other hard-coded '50' a few lines above. It would probably be best to create a constant for that value.